lib/stdlib/src/sets.erl

```diff
@@ -74,7 +82,7 @@
 
 -type set() :: set(_).
```

Comment:
I know it's probably outside the scope of this pull request, but would it make sense to make this type opaque, too?

Reply:
Would't this mean that dialyzer couldn't unify them? So if you had a function returning `set()` you wouldn't be able to pass the result to another function expecting `set(_)` as an argument?

Reply:
I'm unaware if that would be the case, but would like to understand the same as you. I can try it locally though, to see if it makes sense. If what you ask is replied in the sense of "Yes, dialyzer couldn't unify them", then my request is no good and can be scratched.

Reply:
You are right 😄. It breaks opacity, for `dialyzer`. In any case (not the scope of this PR, surely) I wonder if that should be the case, though.